taxonomy-editor-frontend/src/pages/gotoproject/index.jsx

```diff
@@ -6,7 +6,7 @@ import { API_URL } from "../../constants";
 import MaterialTable from "@material-table/core";
 import EditIcon from "@mui/icons-material/Edit";
 import { useEffect } from "react";
-import { toSnakeCase, toTitleCase } from "../../components/interConvertNames";
+import { toSnakeCase, toTitleCase } from "../../utils";
```

Comment:
Please make sure that the imports are updated everywhere.

Reply:
double checked this, and everything cool